package.json

```diff
@@ -14,7 +14,7 @@
     "test": "echo \"No test specified\"",
     "release": "np --no-2fa",
     "start": "ts-node ./examples/index.ts",
-    "lint": "eslint --fix .",
+    "lint": "eslint --max-warnings 0 .",
```

Comment:
We can keep fix here - this makes sense for the package.json because the package.json is used locally and the person would want to fix their errors using this command.

But we want to use this `--max-warnings` in the .circleci.

Reply:
It seems likely that the person running lint wants their linting errors to be fixed.